tests/detectors/test_feature_detector.py

```diff
@@ -11,35 +11,40 @@
 from os.path import abspath
 
 from preggy import expect
-from tests.base import DetectorTestCase
+from tornado.testing import gen_test
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'tornado.testing'](https://app.codacy.com/manual/heynemann/thumbor/pullRequest?prid=4967909)